cmd/kubeadm/app/images/images_windows.go

```diff
@@ -30,5 +30,5 @@ func GetPauseImage(cfg *kubeadmapi.ClusterConfiguration) string {
 	if cfg.ImageRepository != kubeadmapiv1beta2.DefaultImageRepository {
 		return GetGenericImage(cfg.ImageRepository, "pause", constants.PauseVersion)
 	}
-	return GetGenericImage("mcr.microsoft.com/oss/kubernetes", "pause", constants.PauseVersion)
+	return GetGenericImage("k8s.gcr.io", "pause", constants.PauseVersion)
```

Comment:
we should remove this file and the one for images_unix.go.

basically now that the image location is the same the images_unix.go function GetPauseImage can live under images.go.